### ISSUE TITLE
Volume's options option seems to break idempotency

### ISSUE THREAD

**Issue by jaudriga** [containers/ansible-podman-collections#108]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

When using the `podman_volume` module with the `options` option, the string seems to break idempotency.
```
    options: "device={{ data_device }}"
```

**Steps to reproduce the issue:**

1. Run the following:
```
---
- hosts: all
  tasks:
    - name: Create test volume on external volume
      containers.podman.podman_volume:
        name: test
        options: "/dev/nvme3n1"
```

2. Run it again.

**Describe the results you received:**

Volume gets recreated on every run.

**Describe the results you expected:**

Volume gets created only once.

**Additional information you deem important (e.g. issue happens only occasionally):**

Maybe the string `device=something` is not being properly parsed? It is also weird that one has to specify `device=` at all.

**Output of `ansible --version`:**

```
ansible 2.9.13
  config file = /home/usr/repos/audriga-infra/ansible.cfg
  configured module search path = ['/home/usr/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.8/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.8.5 (default, Sep  5 2020, 10:50:12) [GCC 10.2.0]
```

**Output of `podman version`:**

```
Version:      2.0.5
API Version:  1
Go Version:   go1.14.2
Built:        Thu Jan  1 00:00:00 1970
OS/Arch:      linux/amd64
```

**Output of `podman info --debug`:**

```
host:
  arch: amd64
  buildahVersion: 1.15.1
  cgroupVersion: v1
  conmon:
    package: 'conmon: /usr/libexec/podman/conmon'
    path: /usr/libexec/podman/conmon
    version: 'conmon version 2.0.20, commit: '
  cpus: 2
  distribution:
    distribution: ubuntu
    version: "20.04"
  eventLogger: file
  hostname: hostname
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  kernel: 5.4.0-1022-aws
  linkmode: dynamic
  memFree: 150753280
  memTotal: 4064808960
  ociRuntime:
    name: runc
    package: 'cri-o-runc: /usr/lib/cri-o-runc/sbin/runc'
    path: /usr/lib/cri-o-runc/sbin/runc
    version: 'runc version spec: 1.0.2-dev'
  os: linux
  remoteSocket:
    path: /run/user/1000/podman/podman.sock
  rootless: true
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: 'slirp4netns: /usr/bin/slirp4netns'
    version: |-
      slirp4netns version 1.1.4
      commit: unknown
      libslirp: 4.3.1-git
      SLIRP_CONFIG_VERSION_MAX: 3
  swapFree: 0
  swapTotal: 0
  uptime: 166h 36m 39.73s (Approximately 6.92 days)
registries:
  search:
  - docker.io
  - quay.io
store:
  configFile: /home/ubuntu/.config/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: vfs
  graphOptions: {}
  graphRoot: /home/ubuntu/.local/share/containers/storage
  graphStatus: {}
  imageStore:
    number: 0
  runRoot: /run/user/1000/containers
  volumePath: /home/ubuntu/.local/share/containers/storage/volumes
version:
  APIVersion: 1
  Built: 0
  BuiltTime: Thu Jan  1 00:00:00 1970
  GitCommit: ""
  GoVersion: go1.14.2
  OsArch: linux/amd64
  Version: 2.0.5
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman/unknown 2.0.6~1 amd64 [upgradable from: 2.0.5~2]
podman/unknown 2.0.6~1 arm64
podman/unknown 2.0.6~1 armhf
podman/unknown 2.0.6~1 s390x
```

**Playbok you run with ansible (e.g. content of `playbook.yaml`):**

See above

**Command line and output of ansible run with high verbosity:**

```
$ ansible-playbook -v test.yml 
Using /home/usr/repos/repo/ansible.cfg as config file

PLAY [all] ***********************************************************************************************************************************************************************************

TASK [Gathering Facts] ***********************************************************************************************************************************************************************
ok: [testhost]

TASK [Create test volume on external volume] *************************************************************************************************************************************************
changed: [testhost] => {"actions": ["created test"], "changed": true, "podman_actions": ["podman volume create test --opt device=/dev/nvme3n1"], "stderr": "", "stderr_lines": [], "stdout": "test\n", "stdout_lines": ["test"], "volume": {"Anonymous": false, "CreatedAt": "2020-09-08T06:11:36.195055342Z", "Driver": "local", "GID": 0, "Labels": {}, "Mountpoint": "/var/lib/containers/storage/volumes/test/_data", "Name": "test", "Options": {"device": "/dev/nvme3n1"}, "Scope": "local", "UID": 0}}

PLAY RECAP ***********************************************************************************************************************************************************************************
testhost               : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

$ ansible-playbook -v test.yml 
Using /home/usr/repos/audriga-infra/ansible.cfg as config file

PLAY [all] ***********************************************************************************************************************************************************************************

TASK [Gathering Facts] ***********************************************************************************************************************************************************************
ok: [testhost]

TASK [Create test volume on external volume] *************************************************************************************************************************************************
changed: [testhost] => {"actions": ["recreated test"], "changed": true, "podman_actions": ["podman volume rm -f test", "podman volume create test --opt device=/dev/nvme3n1"], "stderr": "", "stderr_lines": [], "stdout": "test\n", "stdout_lines": ["test"], "volume": {"Anonymous": false, "CreatedAt": "2020-09-08T06:11:49.197938207Z", "Driver": "local", "GID": 0, "Labels": {}, "Mountpoint": "/var/lib/containers/storage/volumes/test/_data", "Name": "test", "Options": {"device": "/dev/nvme3n1"}, "Scope": "local", "UID": 0}}

PLAY RECAP ***********************************************************************************************************************************************************************************
testhost               : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
AWS

**Comment by sshnaidm** [comment]
@jaudriga I configured options as a list in a new PR because we can set a few options according to the [manual](https://github.com/containers/podman/blob/master/docs/source/markdown/podman-volume-create.1.md). Please check if it solves your issue. But need to set options as a list in tasks like:
```yaml
- podman_volume:
    state: present
    name: vol
    options:
      - "device=/dev/loop1"
      - "type=ext4"
```

**Comment by jaudriga** [comment]
@sshnaidm it works. Thanks!
